connection.go

```diff
@@ -258,6 +267,10 @@ func (mc *mysqlConn) interpolateParams(query string, args []driver.Value) (strin
 }
 
 func (mc *mysqlConn) Exec(query string, args []driver.Value) (driver.Result, error) {
+	return mc.ExecContext(backgroundCtx(), query, args)
+}
+
+func (mc *mysqlConn) ExecContext(ctx mysqlContext, query string, args []driver.Value) (driver.Result, error) {
```

Comment:
The interface defined in `sql/driver`  is `ExecContext(ctx context.Context, query string, args []NamedValue) (Result, error)`

note the `driver.Value` is changed to `driver.NamedValue`